packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -3168,4 +3168,28 @@ void main() {
     expect(controller.dispatchedPointerEvents, hasLength(1));
     expect(controller.dispatchedPointerEvents[0], isA<PointerHoverEvent>());
   });
+
+  testWidgets('HtmlElementView can be instantiated', (WidgetTester tester) async {
+    late final Widget htmlElementView;
+    expect(() {
+      htmlElementView = const HtmlElementView(viewType: 'webview');
+    }, returnsNormally);
+
+    await tester.pumpWidget(
+      Center(
+        child: SizedBox(
+          width: 100,
+          height: 100,
+          child: htmlElementView,
+        ),
+      )
+    );
+    await tester.pumpAndSettle();
+
+    // This file runs on non-web platforms, so we expect `HtmlElementView` to
+    // fail.
```

Comment:
Instead of this, why not `skip: kIsWeb` on the test? That way we know for sure this is a non-web test only?

Reply:
The entire file is not supposed to run on web (it has `@TestOn('!chrome')`). I bet all tests will fail, including the new one, if we accidentally run this file on web.

The comment is just to clarify for anyone who jumps directly to this test without looking at the entire file.

Reply:
Ah, if there's a `@TestOn('!chrome')`, ship it (`dart-vm` would be better, though, this test would attempt to run in `firefox` if [I'm understanding this right](https://pub.dev/packages/test#platform-selectors))

Reply:
Oh interesting. Yeah we definitely need to fix that. I'll create an issue so we don't forget.